server/db-models/db-models.js

```diff
@@ -0,0 +1,61 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+/* ******************** DB Connection ******************** */
+
+// Connection URL
+const url = `mongodb://heroku_4n2clldv:j3e80jsh2tes5b7r970bejgtq4@ds245927.mlab.com:45927/heroku_4n2clldv`;
+
+//Connection options
+const options = {
+  keepAlive: 1, 
+  connectTimeoutMS: 30000, 
+  reconnectTries: 30, 
+  reconnectInterval: 5000,
+  useNewUrlParser: true
+};
+
+mongoose.connect(url, options);
+
+/* ******************** Schema Definitions ******************** */
+const userSchema = new Schema({
+  username: String,
+  phone_number: {
+    type: Number,
+    min: 10,
+    max: 10
```

Comment:
The min/max options ensure the number falls in a bounded range—not a digit count [see docs](https://mongoosejs.com/docs/schematypes.html#number-validators)

So here, we can trust server-side validation and assume all numbers passed in will be 10 digits